### DISCUSSION TITLE
Added a Repository and it seems to be tracking my entire computer!

### DISCUSSION THREAD

**Discussion by dazehead** [community/community#46695] [New to GitHub] [answered]
I seem to have somehow initiated my new github repository the wrong way as you can see from below. I am just now learning how to use Git. One thing to note is I seem to never be able to get out of (main) I think I have asked git hub to track my entire computer but I dont know. I have tried to look up videos and the github help section. This is my 4th repository and the first time I have seen this. Any advise would be appreciated. 
here is what I am getting when I open VS Code:
![VS_code_error](https://user-images.githubusercontent.com/118700425/217430328-913ea368-a637-4fc6-99ab-d5117e1555a0.jpg)
Here is what I am getting using GitBash
![git_status_error](https://user-images.githubusercontent.com/118700425/217429868-7f5bb55c-c5f8-4785-9eae-240343559bc5.jpg)

**Comment by Juanescacha** [comment, marked as answer]
Sure, thats easy let me help u with that!

If you accidentally ran git init on the wrong path, you can simply remove the Git repository by deleting the .git directory in that path.

to delete it:

Open the terminal and navigate to the directory where you mistakenly initialized Git.

Run the following command:

```bash
rm -rf .git
```
This will delete the .git directory and all of its contents, effectively removing the Git repository from that directory.

Alternatively, you can also use a GUI file explorer to delete the .git directory. Just make sure that hidden files are visible in your file explorer.

remember to Initializate a Git repository in the path that is inside the folder of that project (if you're doing this and also having files outside that path getting tracked I recommend reseting to default git configuration)

You can reset the global Git default configuration by removing the ~/.gitconfig file in your home directory. This will erase all of your Git global settings, including your user name and email, global ignore patterns, and any custom aliases you may have set.

gits is quite simple so do not worry, somethings this errors seems really though but its just bad lucky.

if you have any doubts or I did'nt help u let me know!

**Reply by iiTzRajesh** [reply]
Had the same issue and problem solved, thank you!

**Reply by Maharshibhatnagar** [reply]
Thanks !! i had the same issue

**Reply by afgomz** [reply]
I solve the issue, huge thanks my man

**Reply by bhagwan48** [reply]
Thank u very much! it is working well. Thank u

**Reply by mishrasatyapriya9** [reply]
In windows laptop you can try            rm .git

**Comment by mishrasatyapriya9** [comment]
But I can't't remind in which path I just did the git init , I had some issues in GitHub so I started doing git init in many places so I proporly didn't remember that in which path I just did the git now .
Now it
![Uploading 17098468438951099705459542906069.jpg…]()
 showing 19000 changes in my folders and files because i think it just initialise in global level in my laptop and i couldn't fix it through last 10 days .
Please anyone help me

**Reply by mishrasatyapriya9** [reply]
![17098469251304711117763767822843](https://github.com/community/community/assets/106869525/154c0366-f067-4b14-aaab-797f3bd99cef)

**Reply by Aryanchaudhary4760** [reply]
(git restore -s@ -SW -- . )use this
